pom.xml

```diff
@@ -53,6 +53,11 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20170516</version>
+        </dependency>
```

Comment:
This shouldn't be necessary anymore as we use jackson

Reply:
Ok, I'll need to change some stuff that relies on this, but np.